### PR TITLE
tweak styles for minimal input dropdown when open

### PR DESCRIPTION
**Pull request by mmmmmmaya**
### Summary
- **What:** Fix styles for inputdropdown when menu is open
- **Why:** I missed that there was a separate state for this when implementing
- **Env:** http://localhost:6006/?path=/story/inputdropdown--minimal

#### Before: 
before

#### After: 
after

### Checklist
- [X] I merged latest `main`
- [x] I manually verified the change
- [x] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/core/InputDropdown/styles.ts [hunk 1]
```diff
@@ -72,21 +73,35 @@ const minimal = (props: InputDropdownProps): SerializedStyles => {
...
       }
     }
   `;
 };
+
+const isOpen = (props: InputDropdownProps): SerializedStyles => {
+  const palette = getPalette(props);
+  return css`
+    span {
+      color: ${palette?.text?.primary};
```
Comment (phoenixAja) [inline src/core/InputDropdown/styles.ts:86]: is this color black?

**Review by phoenixAja — APPROVED** [review]
looks good, the only thing is that i want to verify we're using the correct colors as reflected in the design (it looked like the carat and text were both using the same color here)

File: src/core/InputDropdown/styles.ts (hunk 1 above)
Comment (mmmmmmaya) [reply src/core/InputDropdown/styles.ts:86]: the designs actually are not consistent within the design system ref doc. but for [inputdropdown](https://www.figma.com/file/EaRifXLFs54XTjO1Mlszkg/Science-Design-System-Reference?node-id=1876%3A21139), the user input state shows them both as black (the carat should only be colored on active). For [filters](https://www.figma.com/file/EaRifXLFs54XTjO1Mlszkg/Science-Design-System-Reference?node-id=3139%3A19517) (which, iirc, should make use of the inputdropdown, so they should be the same).

I'm checking in with SDS about how the inputdropdown component should actually style it in the default application (then aspen will get it for free).

**Review by tihuan — APPROVED** [review]
LGTM!! Thanks for improving the component 🎉 !

**Pull request merged**